content/post/guides-for-contributors.md

```diff
@@ -52,13 +54,29 @@ The followings are the steps:
 
 Once you create a pull request, the editorial process will be openly conducted within the pull request. Our editors will check the contents and may make some edits.  If we think necessary, we may ask you to edit the post. In that case, you pull the update to your branch, edit the file, commit the changes, and then push the update to the repository. When you and our editors agree the contents, the pull request is merged and your article is published.
 
+#### Method 1-b: PR method (with rmarkdown)
```

Comment:
Wouldn't this be our preferred method? It's easier to include run code inside an rmarkdown file (and I assume most posts will contain at least some code chunks). If that's true, I would change the order of Method 1 and Method 1-b (and call them Method 1a and Method 1b).

Reply:
`.Rmd` should be preferred, and _required_ if there is any executable code. We should provide a template with the header information and code blocks that the user only needs to edit. 

We should also explain how they can fork the repo in order to modify the template, and submit their contribution.

I think the `README` can have these instructions but probably the best place for them, and all of our policies, code of conduct for contributors, etc should use the Wiki functionality for this repo.